src/openforms/logging/tests/test_events.py

```diff
@@ -33,3 +36,58 @@ def test_submission_start(self):
 
             log = TimelineLogProxy.objects.last()
             self.assertEqual(_("Anonymous user"), log.fmt_user)
+
+    def test_submission_logic_evaluated(self):
+
+        submission = SubmissionFactory.from_data(
+            {
+                "firstname": "foo",
+                "birthdate": "2022-06-20",
+            },
+        )
+
+        json_logic_trigger = {
+            ">": [{"date": {"var": "birthdate"}}, {"date": "2022-06-20"}]
+        }
+
+        json_logic_trigger_2 = {"==": [{"var": "firstname"}, "bar"]}
+
+        rule = FormLogicFactory(
+            form=submission.form,
+            json_logic_trigger=json_logic_trigger,
+            actions=[],
+        )
+        rule_2 = FormLogicFactory(
+            form=submission.form,
+            json_logic_trigger=json_logic_trigger_2,
+            actions=[],
+        )
+
+        logevent.submission_logic_evaluated(
+            submission,
+            [{"rule": rule, "trigger": True}, {"rule": rule_2, "trigger": False}],
+            submission.get_merged_data(),
```

Comment:
```suggestion
            submission.data,
```
we prefer the property accessor instead of calling the method :)